test_conformance/api/test_kernels.cpp

```diff
@@ -87,6 +87,24 @@ const char *sample_two_kernel_program[] = {
 "\n"
 "}\n" };
 
+const char *sample_read_only_image_test_kernel = {
```

Comment:
We should use C++ raw string literals for new code.

For an example, see: https://github.com/KhronosGroup/OpenCL-CTS/blob/d918b4165763a55d831a37efc8aea35eb5a09fdc/test_conformance/basic/test_writeimage.cpp#L32-L34

Reply:
Done